addons/auth_signup/views/auth_signup_login.xml

```diff
@@ -20,7 +20,7 @@
             <div class="form-group field-name">
                 <label for="name" class="control-label">Your Name</label>
                 <input type="text" name="name" t-att-value="name" id="name" class="form-control" placeholder="e.g. John Doe"
-                    required="required" t-att-readonly="'readonly' if only_passwords else None"
+                    required="required" t-att-readonly="'readonly' if (only_passwords and name != login) else None"
```

Comment:
here i allowed user to change name if name and log in are same because we inviting user just by email id so we are going provide a way for changing a name when user accept the invitation.